streams/quickstart/java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -47,41 +47,21 @@
         </repository>
     </repositories>
 
-    <!--
-		Execute "mvn clean package -Pbuild-jar"
-		to build a jar file out of this project!
-	-->
-
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.1</version>
+                <version>3.13.0</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
It would be good to set the release flag to 11 - that's the recommended way to do this kind of thing from Java 9 onwards.